New Text Document.txt

```diff
@@ -0,0 +1,166 @@
+function removeMiddle(words) {
+  // words is an array which contains an odd number of strings
+  // return a new array containing only the middle word
+  // the words array should no longer contain the middle word
+  // hint: splice
+  newWord = words.splice(2, 1);
+  return newWord;
+}
+
+function get2ndAnd3rd(myArray) {
+  // myArray is an array of numbers
+  // return an array containing the 2nd and 3rd items from myArray
+  // myArray should remain unchanged
+  // hint: slice
+  newArray = myArray.slice(1, 3);
+  return newArray;
+}
+
+function mapper(myArray) {
+  // myArray is an array of numbers
+  // return a new array which has all items in myArray incremented by one
+  // myArray should remain unchanged
+  // hint: map
+  var newArray = myArray.map(function (i) {
+    return i + 1;
+  });
+  return newArray;
+}
+
+function wordLengths(words) {
+  // words is an array of strings
+  // return a new array that contains the number of letters in each word
+  // hint: strings have a 'length' property
+  var newArr = [];
+  for (i = 0; i < words.length; i++) {
+    newArr[i] = words[i].length;
+  };
+  return newArr;
+}
+
+function cities( capitals, transform ){
+  // capitals is an array of objects that have a city and country property
+  // for example
+  // {
+  //   city: 'Paris',
+  //   country: 'France'
+  // }
+  // formatter is a function that transforms a capital object into a sentence returns it
+  // such as 'Paris is the capital of France'
+
+  // apply formatter to each object in capitals array and return an array of resulting sentences
+//   var newArr = [];
+//   for (i = 0; i < capitals.length; i++){
+//     newArr[i] = capitals.push(transform);
+//   };
+//   return newArr;
+// }
+
+function largerThanTen( numbers ){
+  // numbers is an array of numbers
+  // return a new array that contains only numbers from the input array which are greater than 10
+  // hint: filter
+}
+
+function even( numbers ){
+  // numbers is an array of numbers
+  // return a new array that contains only even numbers from the input array
+  // hint: you may want to use the modulus operator '%'
+}
```

Comment:
Try to avoid using `==` as it is capable of implicit type conversion and so may not give you the result you expect. Unless you have a very good reason, always use the strict equality operator, `===`